apps/deploy-web/src/components/layout/Layout.tsx

```diff
@@ -102,42 +87,58 @@ const LayoutApp: React.FunctionComponent<Props> = ({ children, isLoading, isUsin
   };
 
   return (
-    <>
-      <WelcomeModal open={isShowingWelcome} onClose={onWelcomeClose} />
-
-      <div className="h-full">
-        <div className="h-full w-full" style={{ marginTop: `${ACCOUNT_BAR_HEIGHT}px` }}>
-          <div className="h-full">
-            <Nav isMobileOpen={isMobileOpen} handleDrawerToggle={handleDrawerToggle} />
-
-            <div className="block h-full w-full flex-grow rounded-none md:flex">
-              <Sidebar onOpenMenuClick={onOpenMenuClick} isNavOpen={isNavOpen} handleDrawerToggle={handleDrawerToggle} isMobileOpen={isMobileOpen} />
-
-              <div
-                className={cn("ease ml-0 h-full flex-grow transition-[margin-left] duration-300", {
-                  ["md:ml-[240px]"]: isNavOpen,
-                  ["md:ml-[57px]"]: !isNavOpen
-                })}
-              >
-                {isLoading !== undefined && <LinearLoadingSkeleton isLoading={isLoading} />}
-
-                <ErrorBoundary FallbackComponent={ErrorFallback}>
-                  {!isUsingSettings || isSettingsInit ? (
-                    !isUsingWallet || isWalletLoaded ? (
-                      <div className={cn({ ["container pb-8 pt-4"]: !disableContainer }, containerClassName)}>{children}</div>
-                    ) : (
-                      <Loading text="Loading wallet..." />
-                    )
+    <div className="h-full">
+      {/* Banner for Free Trial and Credit Card payments */}
+      {hasCreditCardBanner && (
+        <>
+          <div className="fixed top-0 flex h-[40px] w-full items-center justify-center space-x-4 bg-primary px-3 py-2">
+            <span className="text-sm font-semibold text-white">Credit Card payments are now available!</span>
+
+            {!hasManagedWallet && <ConnectManagedWalletButton className="mb-2 mr-2 w-full md:mb-0 md:w-auto" size="sm" />}
+          </div>
+          <style jsx global>{`
```

Comment:
is this styling needed? Flex would do it's job on a permanent basis in a more elegant way

Reply:
True, didn't think of that!